parm/config/gefs/config.ufs

```diff
@@ -433,27 +447,30 @@ fi
 
 # Set the name of the UFS (previously nems) configure template to use
 # Default ufs.configure templates for supported model configurations
+if [[ "${USE_ESMF_THREADING:-}" == "YES" ]]; then
+  tmpl_suffix="_esmf"
+fi
 case "${model_list}" in
   atm)
-    default_template="${PARMgfs}/ufs/ufs.configure.atm.IN"
+    default_template="${HOMEgfs}/parm/ufs/ufs.configure.atm${tmpl_suffix:-}.IN"
```

Comment:
Shouldn't all these retain `${PARMgfs}` over `${HOMEgfs}/parm`?

Reply:
yes. done in e7e2b9a3